pypeit/tests/test_flux.py

```diff
@@ -78,7 +78,7 @@ def test_find_standard():
     std_dict = flux.find_standard_file(std_ra, std_dec)
     # Test
     assert std_dict['name'] == 'G191B2B'
-    assert std_dict['calibfile'] == '/data/standards/calspec/g191b2b_mod_005.fits'
```

Comment:
I just changed calibfile to cal_file to fix the 8 character limit of the header.